lib/modules/datasource/docker/index.spec.ts

```diff
@@ -418,9 +418,12 @@ describe('modules/datasource/docker/index', () => {
         .reply(200, '', { 'docker-content-digest': 'some-digest' });
 
       googleAuth.GoogleAuth.mockImplementationOnce(
-        vi.fn().mockImplementationOnce(() => ({
-          getAccessToken: vi.fn().mockResolvedValue('some-token'),
-        })),
+        // TODO: fix typing
```

Comment:
can we capture an Issue for this please? (Doesn't have to have the Issue reference committed into the files, in this case)